spring-xd-integration-test/src/test/java/org/springframework/xd/integration/test/KafkaTests.java

```diff
@@ -35,10 +38,13 @@
 	@Test
 	public void kafkaSourceSinkTest() {
 		String data = UUID.randomUUID().toString();
-		String topic = UUID.randomUUID().toString();
+		String topicToUse = UUID.randomUUID().toString();
+		ZkClient zkClient = sources.getKafkaZkClient();
+		AdminUtils.createTopic(zkClient, topicToUse, 1, 1, new Properties());
+
 		String sinkStreamName = "ks" + UUID.randomUUID().toString();
-		KafkaSource source = sources.kafkaSource().topic(topic).ensureReady();
-		KafkaSink sink = sinks.kafkaSink().topic(topic).ensureReady();
+		KafkaSource source = sources.kafkaSource().topic(topicToUse).ensureReady();
```

Comment:
tried it out, because of the api chaining, prob best to refactor common code into a util class can call it from within KafkaSource/Sink, playing around some more